mqtt/src/main/scala/akka/stream/alpakka/mqtt/javadsl/MqttSource.scala

```diff
@@ -7,31 +7,72 @@ package akka.stream.alpakka.mqtt.javadsl
 import java.util.concurrent.CompletionStage
 
 import akka.Done
-import akka.stream.alpakka.mqtt.{MqttMessage, MqttSourceSettings}
+import akka.stream.alpakka.mqtt._
+import akka.stream.javadsl.Source
 
+import scala.compat.java8.FutureConverters._
+
+/**
+ * Java API
+ *
+ * MQTT source factory.
+ */
 object MqttSource {
 
   /**
-   * Java API: create an [[MqttSource]] with a provided bufferSize.
+   * Create a source subscribing to MQTT messages (without a commit handle).
+   *
+   * The materialized value completes on successful connection to the MQTT broker.
+   *
+   * @param bufferSize max number of messages read from MQTT before back-pressure applies
+   * @deprecated use atMostOnce with MqttConnectionSettings and MqttSubscriptions instead
+   */
+  @deprecated("use atMostOnce with MqttConnectionSettings and MqttSubscriptions instead", "0.21")
```

Comment:
Just a reminder that we will need to grep those version values and replace to the first milestone version before we release the milestone.